uaclient/jobs/license_check.py

```diff
@@ -4,39 +4,38 @@
 """
 import logging
 
-from uaclient import config, exceptions
+from uaclient import config, exceptions, jobs, util
 from uaclient.cli import action_auto_attach, setup_logging
 from uaclient.clouds.gcp import GCP_LICENSES, UAAutoAttachGCPInstance
 from uaclient.clouds.identity import get_cloud_type
+from uaclient.config import UAConfig
```

Comment:
No need to import this - `config.UAConfig` is already being used for the typing.

Reply:
(shows outdated because of the next commit, but it's still there, I swear!

Reply:
Ah! You're totally right, my bad